graylog2-server/src/main/java/org/graylog2/shared/security/RestPermissions.java

```diff
@@ -150,6 +150,10 @@ public class RestPermissions implements PluginPermissions {
     public static final String TYPE_MAPPINGS_DELETE = "typemappings:delete";
     public static final String TYPE_MAPPINGS_EDIT = "typemappings:edit";
     public static final String TYPE_MAPPINGS_READ = "typemappings:read";
+    public static final String PROFILES_CREATE = "profiles:create";
```

Comment:
I think `profiles` is way too generic. We should name it `typeprofiles` or `mappingprofiles`.

Reply:
Done!